sessions/slides/forecasting-as-an-epidemiological-problem.qmd

```diff
@@ -5,56 +5,81 @@ format:
+- how many beds to we need?
+- where should we allocate vaccines?
+- where should we trial vaccines?
 
-# Why forecast?
+However:
+
+- not all modelling is prediction
+- not all modelling of the future is forecasting
+
+### Different ways of modelling the future {.smaller}
+
+- **Nowcasts** make statements about current trends based on partial data
+
+- **Forecasts** are *unconditional* statements about the future: what *will happen*
+
+- **Scenarios** state what *would* happen under certain conditions
+
+### Why nowcast/forecast? {.smaller}
 
 - to create **situational awareness**
   - nowcast: where are we now?
   - forecast: where are we heading?
 
-# Example uses of forecasts
+### CDC use of influenza forecasts
+
+![](figures/cdc_flu_forecast_use.png)
+
+[CDC: About Flu Forecasting](https://www.cdc.gov/flu/weekly/flusight/how-flu-forecasting.htm)
+
+### Relationship with $R_t$ estimation {.smaller}
+
+- In infectious disease epidemiology, "no change" means no change in $R_t$
+- In that sense, predicting infections comes down to predicting $R_t$
+- Commonly, forecast models assume no change in $R_t$. Is this a good assumption?
+
+### Relationship with nowcasting {.smaller}
 
-- healthcare capacity planning
-- vaccine trial planning
+- Nowcast: we have **some** data from the dates we're predicting
+- Forecast: we have **no** data from the dates we're predicting (usually: because they're in the future)
 
-# Relationship with R estimation
+### Forecasts are usually *probabilistic* {.smaller}
 
-- Commonest assumption is constant R ("no change")
-- Is this a good forecast model?
+- Because the future is uncertain, it is natural to express predictions in probabilities, e.g. there is a X% chance of exceeding Y hospital admissions.
 
-# Difference to nowcast
+![](figures/weather_forecast.png)
 
-- model takes over from data
+### Importance of evaluation {.smaller}
 
-# Importance of evaluation
+- Because forecasts are *unconditional* (what will happen) we can compare them to data and see how well they did
+- So-called **proper scoring rules** incentivise forecasters to express an honest belief about the future
+- Many proper scoring rules (and other metrics) are available to assess probabilistic forecasts
 
-- we can score forecasts (because they're unconditional) and nowcasts; this is much harder for conditional forecasts (are the conditions met)?
+### Forecasting paradigm {.smaller}
 
-# Importance of uncertainty
+Maximise *sharpness* subject to *calibration*
 
-- how certain we are about certain future outcomes is important
-- providing uncertainty allows attributing probabilities to outcomes, i.e. X% chance of exceeding hospital capacity
-- Makes evaluation more complicated (but not impossible)
+- Statements about the future should be **correct** ("calibration")
+- statements about the future should aim to have **narrow uncertainty** ("sharpness")
 
-# Plan for the next few sessions
+### Plan for this two sessions
```

Comment:
```suggestion
### Plan for the next two sessions
```

Reply:
actually meant to say "plan for this session"